### PR TITLE
ported lite-node tutorial fixes from docs.filecoin.io

### PR DESCRIPTION
**Pull request by smagdali**
This is essentially the same set of changes as https://github.com/filecoin-project/filecoin-docs/pull/1311, 

- updates to 1.16.1
- adds build-from-source instructions for mac, as Homebrew is not currently building reliably.
- removes AppImage from Linux, pending CI fix
- clarifies and corrects Snap installation instructions, including not running a full node, and the difference in binary name.
- better explanation of the relationship between FIL and datacap.

It is not perfect. Until we have builds working better and more reliably, installing lotus is going to be tricky. It is definitely better than what is currently on the site, and I intend to continue to iterate on it.

### CODE REVIEW

File: content/en/tutorials/lotus/store-and-retrieve/set-up.md [hunk 1]
```diff
@@ -111,15 +106,71 @@ You can install Lotus on MacOS 10.11 El Capitan or higher. You must have [Homebr
...
+    ```
+1. Install Rust (when prompted, choose the default 'install' option) 
+    ```shell
+    rustup-init
+    ```
+
+1. Build the clients
+    ```shell
+    make all
+    ```
```
Comment (johnnymatthews) [inline content/en/tutorials/lotus/store-and-retrieve/set-up.md:148]: Make sure there's an empty line on either side of a code block:

````
1. Checkout the latest release:

    ```shell
    git checkout tags/v1.16.1
    ```

1. Setup some environment variables correctly:

    ```shell
    export LIBRARY_PATH=/opt/homebrew/lib; export FFI_BUILD_FROM_SOURCE=1
    ```

````

File: content/en/tutorials/lotus/store-and-retrieve/set-up.md [hunk 2]
```diff
@@ -111,15 +106,71 @@ You can install Lotus on MacOS 10.11 El Capitan or higher. You must have [Homebr
...
+
+{{< alert icon="tip" >}}If you get a warning: 'kIOMasterPortDefault' is deprecated: first deprecated in macOS 12.0' , don't worry - the build still worked. 
+{{< /alert >}}
```
Comment (johnnymatthews) [inline content/en/tutorials/lotus/store-and-retrieve/set-up.md:151]: Does this need to be in an alert? If you over use alerts for this kind of stuff then they lose their impact on the really important sections. Try to use alerts sparingly.

**Review by johnnymatthews — CHANGES_REQUESTED** [review]
A couple of minor changes requested.

File: content/en/tutorials/lotus/store-and-retrieve/set-up.md (hunk 1 above)
Comment (smagdali) [reply content/en/tutorials/lotus/store-and-retrieve/set-up.md]: done

File: content/en/tutorials/lotus/store-and-retrieve/set-up.md (hunk 2 above)
Comment (smagdali) [reply content/en/tutorials/lotus/store-and-retrieve/set-up.md]: I'll take it out if you really disagree.

File: content/en/tutorials/lotus/store-and-retrieve/set-up.md [hunk 3]
```diff
@@ -129,33 +193,38 @@ There are two simple ways to install Lotus on Ubuntu:
...
+{{< alert icon="warning" >}}**Requirements**
 You must have [Snapd](https://snapcraft.io/docs/installing-snapd) installed.
 {{< /alert >}}
 
 1. To install Lotus using Snap, run:
 
     ```shell
-    snap install lotus-filecoin
+    sudo snap install lotus-filecoin
+    ```
```
Comment (johnnymatthews) [inline content/en/tutorials/lotus/store-and-retrieve/set-up.md:223]:
````suggestion
    ```

````

File: content/en/tutorials/lotus/store-and-retrieve/store-data/index.md [hunk 4]
```diff
@@ -237,9 +210,9 @@ Once the data has been sent to the storage clients, the storage deals can take u
     ```shell
     lotus client list-deals --show-failed
     ```
```
Comment (johnnymatthews) [inline content/en/tutorials/lotus/store-and-retrieve/store-data/index.md:212]:
````suggestion
    ```

````

File: content/en/tutorials/lotus/store-and-retrieve/store-data/index.md [hunk 5]
```diff
@@ -237,9 +210,9 @@ Once the data has been sent to the storage clients, the storage deals can take u
...
-
-    **DO NOT TURN OFF YOUR LOTUS NODE!** Your Lotus lite-node needs to remain online until the deal state has reached `StorageDealActive`. See the [Processing states](#processing-states) table below to find out which states happen and when.
-
+{{< alert icon="warning" >}}**DO NOT TURN OFF YOUR LOTUS NODE!** 
+Your Lotus lite-node needs to remain online until the deal state has reached `StorageDealActive`. See the [Processing states](#processing-states) table below to find out which states happen and when.
+{{< /alert >}}
```
Comment (johnnymatthews) [inline content/en/tutorials/lotus/store-and-retrieve/store-data/index.md:215]:
```suggestion
{{< /alert >}}

```

**Review by johnnymatthews — APPROVED** [review]
Looks good from my end. Just double check the preview that everything's looking how it should; then feel free to merge when ready.

File: assets/scss/components/_alerts.scss [hunk 6]
```diff
@@ -24,8 +24,9 @@
 }
 
 .alert-warning {
-  background: $beige;
-  color: $black;
+  color: #664d03;
+  background-color: #fff3cd;
+  border-color: #ffecb5;
```
Comment (johnnymatthews) [inline assets/scss/components/_alerts.scss:29]: What're these changes for? Why aren't you using the variables in [/assets/scss/common/_variables.scss](https://github.com/filecoin-project/lotus-docs/blob/main/assets/scss/common/_variables.scss)?

**Pull request merged**